spread.yaml

```diff
@@ -922,13 +922,11 @@ suites:
             ENABLE_SECURE_BOOT: '$(HOST: echo "${SPREAD_ENABLE_SECURE_BOOT:-false}")'
             # Build and use snapd from current branch
             BUILD_SNAPD_FROM_CURRENT: '$(HOST: echo "${SPREAD_BUILD_SNAPD_FROM_CURRENT:-false}")'
+            # Download and use an custom image from this url
+            CUSTOM_IMAGE_URL: '$(HOST: echo "${SPREAD_CUSTOM_IMAGE_URL:-}")'
             # Directory where the images and test assets are stored
             WORK_DIR: '/tmp/work-dir'
         manual: true
-        debug: |
-            if [ -f "${WORK_DIR}/serial.log" ]; then
-                cat "${WORK_DIR}/serial.log"
-            fi
```

Comment:
why delete this ? if we have this here then when preparing a test fails we still get the boot log, which has proven to be useful numerous times

Reply:
@anonymouse64 Sorry, I though is was needed temporally to see the reboot errors, I already add it again.